packages/dcos-net/extra/dcos-net-setup.py

```diff
@@ -21,6 +21,13 @@ def main():
         if result.stderr.strip().endswith(b'File exists'):
             result.returncode = 0
         sys.exit(result.returncode)
+    if sys.argv[1] == 'iptables':
+        argv = ['-C' if arg in ['-A', '-I'] else arg for arg in sys.argv[1:]]
```

Comment:
So, this would rewrite `dcos-net-setup.py iptables --wait -A FORWARD -j ACCEPT` to `iptables --wait -C FORWARD -j ACCEPT`. Is that correct? Can you add a code comment for why that is?

Reply:
Ok, I now see that the `subprocess.run()` way of "doing things" is for "checking". And if the check succeeds (rule already in place) we exit, and otherwise modify the rules instead via `os.execvp(sys.argv[1], sys.argv[1:])`.

Why didn't you want to use `subprocess.run()` for the second child process invocation?

Reply:
For the second child process we don't need to check the exist status and it's the last call in the script. So `exec(2)` syscall is the most convenient method to run that command in POSIX. If it's not the pythonic way I'll use only `subprocess.run()` in the script.